cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -248,6 +242,12 @@ rules:
   - delete
   - get
   - update
+- apiGroups:
+  - k8s.cni.cncf.io
+  resources:
+  - network-attachment-definitions
```

Comment:
The NAD is a namespace-specific resource, right? If so, I would think this priv could stay in the Role instead of moving it to the ClusterRole. You saw it failed if you didn't move it?

Reply:
If we are fetching NAD in any other namespace, we will need it in ClusterRole.
Else we will have to create Role for NAD every time for each namespace.